### DISCUSSION TITLE
GitHub Mobile set as preferred 2FA verification system but GitHub still uses Authenticator App

### DISCUSSION THREAD

**Discussion by poweredbypigeon** [community/community#125092] [New to GitHub]
I've enabled two factor authentication on my account and installed GitHub Mobile on a mobile device, as I didn't want to download a third party app to scan the QR code. I then set GitHub Mobile as my preferred system to log in.

However, when I try to log in, GitHub gives me a QR code and prompts me for the six-digit code despite GitHub Mobile being my preferred system. Meanwhile, no pop ups occur on my mobile device. Is there any way to disable the Authenticator App configuration or get GitHub to use GitHub Mobile by default, or will I have to download a third party app to log in?

**Comment by Mortiemi** [comment]
> I've enabled two factor authentication on my account and installed GitHub Mobile on a mobile device, as I didn't want to download a third party app to scan the QR code. I then set GitHub Mobile as my preferred system to log in.
> 
> 
> 
> However, when I try to log in, GitHub gives me a QR code and prompts me for the six-digit code despite GitHub Mobile being my preferred system. Meanwhile, no pop ups occur on my mobile device. Is there any way to disable the Authenticator App configuration or get GitHub to use GitHub Mobile by default, or will I have to download a third party app to log in?

**Comment by Anupamit** [comment]
Check this discussion [https://github.com/orgs/community/discussions/10861](url)

**Comment by datsculptor** [comment]
It's possible that GitHub's system isn't fully integrating with GitHub Mobile as expected. You may want to reach out to GitHub support for assistance in troubleshooting this issue.
